src/server/scripts/Spells/spell_druid.cpp

```diff
@@ -1664,10 +1686,9 @@ class spell_dru_travel_form_dummy : public SpellScriptLoader
             if (!player)
                 return SPELL_FAILED_CUSTOM_ERROR;
 
-            if (player->GetSkillValue(SKILL_RIDING) < 75)
-                return SPELL_FAILED_APPRENTICE_RIDING_REQUIREMENT;
+            uint32 spellId = (player->HasSkill(SPELL_DRUID_FORM_AQUATIC) && player->IsInWater()) ? SPELL_DRUID_FORM_AQUATIC : SPELL_DRUID_FORM_STAG;
```

Comment:
`HasSkill(SPELL_DRUID_FORM_AQUATIC)` should be `HasSpell(SPELL_DRUID_FORM_AQUATIC)` ?